DependencyInjection/ONGRElasticsearchExtension.php

```diff
@@ -39,23 +39,71 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('es.connections', $config['connections']);
         $container->setParameter('es.managers', $config['managers']);
 
+        $this->addDocumentFinderDefinition($config, $container);
         $this->addMedadataCollectorDefinition($config, $container);
         $this->addDocumentsResource($config, $container);
         $this->addDataCollectorDefinition($config, $container);
     }
 
     /**
-     * Adds MetadataCollector definition to container.
+     * Adds DocumentFinder definition to container (ID: es.document_finder).
+     *
+     * @param array            $config
+     * @param ContainerBuilder $container
+     */
+    private function addDocumentFinderDefinition(array $config, ContainerBuilder $container)
+    {
+        $documentFinder = new Definition(
+            'ONGR\ElasticsearchBundle\Mapping\DocumentFinder',
+            [
+                $container->getParameter('kernel.bundles'),
+            ]
+        );
+        $documentFinder->addMethodCall('setDocumentDir', [$config['document_dir']]);
```

Comment:
Since it's most likely last big change in the es bundle, its good time to change `document_dir` to something more meaningful? I mean using shortcuts isn't a good example.